README.md

```diff
@@ -12,6 +12,10 @@ Inspired by [tokei](https://github.com/Aaronepower/tokei).
 ## Installation
 
 ```
+# Go version >= 1.16
+$ go install github.com/hhatto/gocloc/cmd/gocloc
```

Comment:
```suggestion
$ go install github.com/hhatto/gocloc/cmd/gocloc@latest
```

Is it necessary to specify the version for installation?

Reply:
Thanks for the fix. I should have added the version of the package.